src/index.js

```diff
@@ -1,10 +1,14 @@
 var parser = require('graphql/language/parser');
+var stripIgnoredCharacters = require('graphql').stripIgnoredCharacters;
```

Comment:
This is not good! When used on frontend, you will add to the bundle size the whole size of `graphql` dependency.

So, 2 ways to fix this:
1. quick, just `var stripIgnoredCharacters = require('graphql/utilities/stripIgnoredCharacters')`. Similar way as graphql parser is included
2. more complicated, rewrite this module into ES6 and use webpack/rollup to shake out unused dependencies in prebuilt version

Reply:
Done (1.).